weather-app/index.js

```diff
@@ -3,47 +3,55 @@ let form = document.getElementById('form');
 let buttonClear = document.getElementById('clear');
 
 let tableBody = document.getElementById('tbody');
+let cityMap = document.querySelector('.cityMap');
 
 const map = new Map();
 
 function renderTableRow() {
 
     let tBody = '';
-
+    let cMap = '';
     map.forEach((value, key) => {
-        const[city, country, temp] = value;
+        const[city, country, temp, lat, lon] = value;
         tBody += `
         <tr>
             <td>${city}</td>
             <td>${country}</td>
             <td>${temp}</td>
             <td>${temp}</td>
         </tr>
+       
         `
+        cMap += `<img src='https://maps.locationiq.com/v2/staticmap?key=pk.c6c8e0f26d559c4194d567e972e29b74&center=${lat},${lon}&zoom=12&size=400x400&markers=${lat},${lon}|icon:large-red-cutout'></img>`
```

Comment:
May be better to add all markers into one static map instead of creating several?